examples/lump-mpi.py

```diff
@@ -105,8 +105,8 @@ def main(ctx_factory=cl.create_some_context):
     nodes = thaw(actx, discr.nodes())
     current_state = initializer(nodes)
 
-    visualizer = make_visualizer(discr, discr.order + 3
-                                 if discr.dim == 2 else discr.order)
+    visualizer = make_visualizer(discr, order + 3
+                                 if discr.dim == 2 else order)
```

Comment:
```suggestion
    visualizer = make_visualizer(discr)
```

Same below.

x-ref: https://github.com/inducer/grudge/issues/65, https://github.com/inducer/grudge/pull/86

Reply:
(c9ea674)